django/contrib/postgres/operations.py

```diff
@@ -15,6 +15,10 @@ def database_forwards(self, app_label, schema_editor, from_state, to_state):
         if schema_editor.connection.vendor != 'postgresql':
             return
         schema_editor.execute("CREATE EXTENSION IF NOT EXISTS %s" % schema_editor.quote_name(self.name))
+        # Register new type handlers straight away as it cannot be done before
```

Comment:
I'm not exactly sure what "straight away" means here. I wonder if something like this could be an improvement:
```
# Registering new type handlers cannot be done before the extension is
# installed, otherwise a subsequent data migration would use the same
# connection.
```